### PR TITLE
Add nodejs-theforeman-builder package

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: comps/comps-foreman-rhel7.xml [hunk 1]
```diff
@@ -121,15 +121,16 @@
       <packagereq type="default">tfm-nodejs-redux</packagereq>
       <packagereq type="default">tfm-nodejs-redux-thunk</packagereq>
       <packagereq type="default">tfm-nodejs-regenerator-runtime</packagereq>
-      <packagereq type="default">tfm-nodejs-reselect</packagereq>
       <packagereq type="default">tfm-nodejs-regenerator-runtime</packagereq>
       <packagereq type="default">tfm-nodejs-reselect</packagereq>
+      <packagereq type="default">tfm-nodejs-reselect</packagereq>
```
Comment (sharvit) [inline comps/comps-foreman-rhel7.xml]: Looks like `tfm-nodejs-reselect` is appearing twice

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
[test rpm]

**Comment by evgeni** [comment]
The following PRs need to be merged "together": #4417 #4416 #4413

However, they are blocked on the discussion in https://github.com/theforeman/foreman/pull/7178

**Pull request merged**